### PR TITLE
Term en/sli slo sla

### PR DESCRIPTION
**Pull request by fsbaraglia**
first version of SLI and SLO, SLA will come asap

### CODE REVIEW

File: content/en/service_level_objective.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Service Level Objective
+status: Feedback appreciapted
+category: Property
+---
+
+Service level objective (SLO) is a target value or range of values for a service level that is measured by an [SLI](/service_level_indicator/). SLO is the objective that must be achieved and it reflects a smooth and successful user journey for the User of a service. 
+
+SLO is the key element for [Site Reliability Engineers](https://glossary.cncf.io/site_reliability_engineering/), [DevOps](https://glossary.cncf.io/devops/) and Business Teams to measure the quality of the user journey. SLO and [Error-budget](https://glossary.cncf.io/error_budget/) can be used to balance the delivery of new features to production. SLOs are targets set by engineering teams from the end-user perspective and not the operational or engineering team point of view.
```
Comment (jihoon-seo) [inline content/en/service_level_objective.md:9]:
```suggestion
SLO is the key element for [Site Reliability Engineers](/site_reliability_engineering/), [DevOps](/devops/) and Business Teams to measure the quality of the user journey. SLO and Error-budget can be used to balance the delivery of new features to production. SLOs are targets set by engineering teams from the end-user perspective and not the operational or engineering team point of view.
```

- Updated to relative URIs
- CNCF Glossary does not have the term `Error-budget` yet. 😊

**Review by jihoon-seo — COMMENTED** [review]
@fsbaraglia Thank you for your contribution. 😊
And PTAL!

File: content/en/service_level_indicator.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Service Level Indicator
+status: Feedback appreciapted
```
Comment (jihoon-seo) [inline content/en/service_level_indicator.md:3]:
```suggestion
status: Feedback appreciated
```

File: content/en/service_level_objective.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: Service Level Objective
+status: Feedback appreciapted
```
Comment (jihoon-seo) [inline content/en/service_level_objective.md:3]:
```suggestion
status: Feedback appreciated
```

**Review by jihoon-seo — COMMENTED** [review]
[Spellcheck CI fails](https://github.com/cncf/glossary/runs/5798523538?check_suite_focus=true#step:4:17).

My suggestion is to add these words to [`wordlist.txt`](https://github.com/cncf/glossary/blob/main/wordlist.txt):
- sli
- slis
- slo
- slos
- qos
- benchmarking

**Review by JasonMorgan — APPROVED** [review]

**Comment by CathPag** [comment]
@fsbaraglia, so...we are currently discussing what should be part of the Glossary and what shouldn't. The CNCF pointed out that terms should fit the [CNCF's cloud native definition](https://github.com/cncf/toc/blob/main/DEFINITION.md) and SLI/SLO/SLA may not fit that definition.

@iamNoah1 @seokho-son @jihoon-seo @JasonMorgan, before Francesco does any more work, can we discuss what to do with these terms?

**Comment by CathPag** [comment]
@fsbaraglia, so sorry for the delay. Unfortunately, we decided not to include these terms in the Glossary because, per our new guidelines, all terms should meet the cloud native definition unless they are foundational to understanding cloud native concepts (e.g. abstraction). I'll go ahead and close this PR.

**Pull request closed without merging**